apps/web/src/views/AIContainerView.tsx

```diff
@@ -0,0 +1,25 @@
+import { WebPreview } from "~features/web-vm/WebPreview"
+
+import dynamic from "next/dynamic"
+
+const WebTerminal = dynamic(
+  () => import("~features/web-vm/WebTerminal").then((m) => m.WebTerminal),
+  {
+    ssr: false
+  }
+)
+
+// Create a screen writing mobile application with a novel UX that no one has seen before, ensure the design is great
+export function AIContainerView() {
+  return (
+    <div className="flex flex-row h-screen bg-slate-1 text-slate-11 p-8 gap-4">
+      <div className="flex flex-col w-2/3 h-full gap-2">
+        <h2 className="text-2xl font-bold text-slate-12">AI Container</h2>
+        <WebTerminal />
+      </div>
+      <div className="flex w-1/3 h-full">
+        <WebPreview />
```

Comment:
I'm never able to see the WebPreview component. screen always looks like this:
<img width="1705" alt="Screenshot 2023-05-07 at 2 34 42 AM" src="https://user-images.githubusercontent.com/1011391/236666904-bad0110e-902a-4f0e-96ff-854635301f6d.png">

Reply:
The Web Preview is only show when there's a server opened and announced by the container :-?... if you do `npm start` on an express script that serves a random page, would it work?